g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/record/service/RecordServiceImpl.java

```diff
@@ -42,7 +40,7 @@
 	}
 
 	@Override
-	public int updateSynchSourceRecord(String product, String version, String component, String locale, int status) {
+	public int updateSynchSourceRecord(String product, String version, String component, String locale, long status) {
```

Comment:
what does 'status' mean?
add description, please!

Reply:
What does 'status' mean, is it a long time? If so, please change the name to be meaningful, e.g.modifiedTime.